engine/src/cubos/engine/window/plugin.cpp

```diff
@@ -1,8 +1,9 @@
 #include <cubos/core/settings.hpp>
 
-#include <cubos/engine/plugins/window.hpp>
+#include <cubos/engine/window/plugin.hpp>
 
 using namespace cubos::core;
```

Comment:
Replace with `using cubos::core::io::Window` etc. The `using namespace cubos::engine` is okay since this file belongs to the engine anyway.